src/cljs/org/broadinstitute/firecloud_ui/main.cljs

```diff
@@ -140,6 +140,42 @@
       (when (:show-nih-link-warning? props)
         [nih-link-warning/NihLinkWarning])])})
 
+;; GAWB-666 Globally show Queued and Cromwell (active) counts
+(defn queue-status-counts [workflow-counts]
+  (let [count-map (get workflow-counts "workflowCountsByStatus" {})
+        queue-time (get workflow-counts "estimatedQueueTimeMS" 0)]
+    {:queue-time queue-time
+     :queued (apply + (map count-map ["Queued" "Launching"]))
```

Comment:
What I mean is that Rawls will never populate `workflowCountsByStatus` with the terminal statuses - so if we do end up using `:terminal` someday, it will be incorrect.  So I would prefer not to have it exist at all.

I agree with what you're saying about consistency - but since the other UI story intentionally differs from Rawls' internal DB states, we can't be consistent here with both.  My opinion is that we should be consistent with the other UI.